packages/legend-extension-dsl-data-space/src/graph-manager/protocol/pure/v1/V1_DSL_DataSpace_PureGraphManagerExtension.ts

```diff
@@ -180,19 +180,19 @@ export class V1_DSL_DataSpace_PureGraphManagerExtension extends DSL_DataSpace_Pu
     const cacheResult = cacheRetriever
       ? await this.fetchDataSpaceAnalysisFromCache(cacheRetriever, actionState)
       : undefined;
-    const engineClient = this.graphManager.engine.getEngineServerClient();
+    const engine = this.graphManager.engine;
     let analysisResult: PlainObject<V1_DataSpaceAnalysisResult>;
     if (cacheResult) {
       analysisResult = cacheResult;
     } else {
       actionState?.setMessage('Fetching project entities and dependencies...');
       const entities = await entitiesRetriever();
       actionState?.setMessage('Analyzing data space...');
-      analysisResult = await engineClient.postWithTracing<
+      analysisResult = await engine.serverClientPostWithTracing<
         PlainObject<V1_DataSpaceAnalysisResult>
       >(
-        engineClient.getTraceData(ANALYZE_DATA_SPACE_TRACE),
-        `${engineClient._pure()}/analytics/dataSpace/render`,
+        engine.serverClientGetTraceData(ANALYZE_DATA_SPACE_TRACE),
+        `${engine.serverClientGetPureBaseUrl()}/analytics/dataSpace/render`,
```

Comment:
Continue the point mentioned in `V1_GraphManagerEngine`, for example, we can just explicitly call 

```ts
getTraceData(..., `${engine.serverClientGetBaseUrl()`/pure/...`)
```